plugins/modules/cargo.py

```diff
@@ -58,15 +58,86 @@
     type: bool
     default: false
     version_added: 7.5.0
+  force:
+    description:
+      - Forcefully install package(s).
+      - Ignores many checks, and updates/installs forcefully.
+    type: bool
+    default: false
+    required: false
+    version_added: 8.2.0
```

Comment:
```suggestion
    version_added: 8.4.0
```
Same for the others.